words2num/lang_ES_US.py

```diff
@@ -0,0 +1,267 @@
+from __future__ import division, unicode_literals, print_function
+import re
+from .core import NumberParseException, placevalue
+from decimal import Decimal, localcontext
```

Comment:
is it possible to separate imports according to pep8 standard?
https://peps.python.org/pep-0008/#imports

Reply:
Thank you. That page says the following-- would the above qualify what it says is ok to do (in bold)?

Imports should usually be on separate lines:
# Correct:
import os
import sys
# Wrong:
import sys, os
It’s okay to say this though:

**# Correct:
from subprocess import Popen, PIPE**

Reply:
Thank you. I'm wondering if it's sort of ok as is?[ __future__](https://www.geeksforgeeks.org/__future__-module-in-python/#:~:text=__future__%20module%20is,the%20previous%20version%20in%20Python.) seems builtin as well as re and they're at the top.

Reply:
Thank you! I pushed this change.